### PR TITLE
Add tests for new version of /users endpoint (confd)

### PR DESCRIPTION
**Pull request by xxbillouxx**
must be reviewed/tested after https://github.com/wazo-platform/wazo-confd/pull/298

### CODE REVIEW

File: features/daily/user_one_call.feature [hunk 1]
```diff
@@ -0,0 +1,55 @@
+Feature: User API - user managed with one HTTP call
+
+
```
Comment (fblackburn1) [inline features/daily/user_one_call.feature:3]: You can follow [STYLEGUIDE](https://github.com/wazo-platform/wazo-acceptance/blob/master/STYLEGUIDE.md)  :)

File: wazo_acceptance/assets/2_users.json [hunk 2]
```diff
@@ -0,0 +1,130 @@
...
+        "uuid": "content_generated_by_code"
+      }
+    ],
+    "switchboards": [
+      {
+        "uuid": "content_generated_by_code"
+      }
+    ]
+  }
+]
```
Comment (fblackburn1) [inline wazo_acceptance/assets/2_users.json:130]: By default all our file have newline at end of line
You can configure your editor

File: features/daily/user_one_call.feature [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+    Given switchboard is "Inn"
+    Given there are users "wazo_acceptance/assets/1_user.json"
+    Given there is an incall "1000@from-extern" to the switchboard "Inn"
+    Given I listen on the bus for "switchboard_queued_calls_updated" messages
+    When incoming call received from "incall" to "1000@from-extern"
+    Then I receive a "switchboard_queued_calls_updated" event
+    Then switchboard "Inn" has "incall" in queued calls
+    When "Rîchard Lâpointe" answer queued call "incall" from switchboard "Inn"
+    Then "Rîchard Lâpointe" is talking to "incall"
+
```
Comment (fblackburn1) [inline features/daily/user_one_call.feature:55]: But no empty line :)

**Review by fblackburn1 — COMMENTED** [review]
Hello, I saw your PR and did a quick feedback ^^

Acceptance test should test a whole workflow, not necessarily corner case, because it have cost to maintain and run
IMHO, I don't see a value to test all incall destinations. Only testing one destination would verify that user is associated to an incall wherever is this destination. Maybe I miss something?

If the thing that you test is the "one HTTP call" it should be in the `When` step
ex:
```diff
    Given there are users "wazo_acceptance/assets/2_users.json"
    When I add a user "wazo_acceptance/assets/2_users.json" with one HTTP call
```

About black:
- to avoid noise, black formatting on code that you don't touch, should be done in a separate commit
- In this repo, I would not blindling reformat this repo. IMHO steps are easier to read without black. But open to discussion about how to standardize this repo or add exception

**Comment by xxbillouxx** [comment]
> 
Hi! 
Thank you for your feedback 🙂

About the fact there are two scenarii for the **incall**:

`Scenario: Incall to destination user` is used to test the incall creation
`Scenario: Incall to users group` is used to test the ring group creation

**Comment by sduthil** [comment]
@xxbillouxx is this PR still useful?
